### DISCUSSION TITLE
extract_tables() return a empty list

### DISCUSSION THREAD

**Discussion by sunday-will** [jsvine/pdfplumber#526] [Ask for help with specific PDFs]
hello!
**problem:**
[here](https://www.renrendai.com/p2p/contract/transfer?type=user&loanId=13552782) is the pdf with two tables,but extract_tables() return a empty list. thx!

**code:**
```
    with pdfplumber.open("xxxx.pdf") as pdf:
        page = pdf.pages[0]
        tables = page.extract_tables()
        print(tables)
```

**environment:**

- pdfplumber version: 0.5.28
- Python version: 3.6.0
- OS:  win 7

**Comment by samkit-jain** [comment]
Hi @sunday-will Appreciate your interest in the library. To extract the tables from the shared PDF, you should use the following table settings
```python
with pdfplumber.open("xxxx.pdf") as pdf:
    page = pdf.pages[0]
    ts = {
        "vertical_strategy": "explicit",
        "horizontal_strategy": "explicit",
        "explicit_vertical_lines": p.curves+p.edges,
        "explicit_horizontal_lines": p.curves + p.edges,
    }
    tables = page.extract_tables(table_settings=ts)
    print(tables)
```

Output:
![icici](https://user-images.githubusercontent.com/15127115/139466210-650f4949-8370-430f-b87e-301ac355826c.png)

**Reply by sunday-will** [reply]
thank you very much @samkit-jain ，it's works well!
